dali/test/python/test_pipeline.py

```diff
@@ -598,15 +598,14 @@ def __init__(self, batch_size, num_threads, device_id):
                                          fill_value = 128,
                                          interp_type = types.INTERP_LINEAR,
                                          use_image_center = True)
-            self.uniform = ops.Uniform(range = (0.0,1.0))
             self.iter = 0
 
         def define_graph(self):
             self.jpegs, self.labels = self.input()
             images = self.decode(self.jpegs)
             outputs = self.cmnp([images, images],
-                                crop_pos_x = self.uniform(),
-                                crop_pos_y = self.uniform())
+                                crop_pos_x = 0.5,
```

Comment:
I think we can remove and use the default.

Reply:
https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops.html#cropmirrornormalize

Reply:
Done.